client/components/layout/NavBar.vue

```diff
@@ -2,43 +2,65 @@
   <article>
     <nav>
       <section>
-      <header><h2>
-        <router-link to="/">Health Journal</router-link>
-      </h2></header>
-      <ul>
-        <li>
-          <router-link to="/">Logs</router-link>
-        </li>
-        <li>
-          <router-link to="/">Trends</router-link>
-        </li>
-      </ul>
+        <header>
```

Comment:
One other comment I forgot to say earlier: in general you should try to avoid having reformatting in the same PR as substantive changes. For example it is hard to tell looking at this diff which parts of this file you acutally changed and which parts were just reformatted. (Am too am guilty of this sometimes, but trying to improve)